src/sass/patterns/_containers.scss

```diff
@@ -1,6 +1,10 @@
 // Containers
 // ==========
 
+.metecho-frame {
+  min-width: 100%;
+}
```

Comment:
Is this overriding a value that comes from SLDS? If so, we might want to clarify that with a comment?